tools/data_source/ncbi_datasets.xml

```diff
@@ -0,0 +1,26 @@
+<tool name="NCBI Datasets Genomes" id="ncbi_datasets_source" tool_type="data_source" version="1.0" profile="21.01">
+    <description>import data from the NCBI Datasets Genomes page</description>
+    <edam_operations>
+        <edam_operation>operation_0224</edam_operation>
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="12.11.0">ncbi-datasets-cli</requirement>
+    </requirements>
+    <version_command>datasets version</version_command>
+    <command><![CDATA[
+datasets download --input-json '$file_param' --filename '$compressed_archive' --no-progressbar
+&& unzip -l '$compressed_archive'
+    ]]></command>
+    <inputs action="https://www.ncbi.nlm.nih.gov/datasets/genomes" check_values="true" method="get">
+        <display>go to NCBI Datasets Genomes server $GALAXY_URL</display>
+        <param name="GALAXY_URL" type="baseurl" value="/tool_runner?tool_id=ncbi_datasets_source"/>
+        <param name="file_param" type="file" optional="true"/>
+    </inputs>
+    <outputs>
+        <data name="compressed_archive" format="zip" label="NCBI Genome Dataset"/>
```

Comment:
@BradHolmes would it make sense to use something like the `dehydrated.zip` format here ? This determines the datatype in Galaxy. If we set this to `dehydrated.zip` the archive would behave exactly like a zip archive, but it would allow downstream tools to say that they only accept `dehydrated.zip` datasets, thereby filtering out other zip archives that might be in the history. So downstream tools don't need to assert that they are really dealing with the correct type of archive and users don't need to be careful in selecting the input dataset.

Reply:
Yes, I think that is a great idea!